validation-test/stdlib/StringSlicesConcurrentAppend.swift

```diff
@@ -20,18 +29,16 @@ extension String {
   }
 }
 
-// Swift.String has an optimization that allows us to append to a shared string
-// buffer.  Make sure that it works correctly when two threads try to append to
-// different non-shared strings that point to the same shared buffer.
-
+// OwnerThread is the thread we expect to end up owning sharedString
+// OtherThread is the thread we expect to *not* own sharedString
```

Comment:
This naming change does not affect the test setup: depending on thread progress, either thread can end up owning the `sharedString`.